assets/js/Embed.js

```diff
@@ -16,14 +26,15 @@ export default class Embed extends React.Component {
     return (
       <div className='embed-wrapper'>
         <div className='embed-not-available'>
-          <h1>This workflow is not available</h1>
+          <h1>{this.props.i18n._(t('workflow.notavailableworkflow')`This workflow is not available`)}</h1>
```

Comment:
There is no specific reason for using prop and no Trans in the corresponding translation (probably because I used prop method in the update section..). I've changed to Trans all the translations in this file, except that one with update because is the only way to achieve our goal